apps/client/src/shared/api/config/end-point.ts

```diff
@@ -17,6 +17,7 @@ export const END_POINT = {
     ) => `posts/${postId}/comments/${commentId}/reply/${commentReplyId}`,
     POST_LIKE: (postId: string) => `posts/${postId}/likes`,
     DELETE_LIKE: (postId: string) => `posts/${postId}/likes`,
+    GET_POPULAR: (size?: number) => `posts/trend?size=${size}`,
```

Comment:
<img width="300" height="62" alt="image" src="https://github.com/user-attachments/assets/c3eb14de-4ccc-4758-8c0f-f64746013395" />

END_POINT는 말 그대로 뒤에 붙는 것들이니

```suggestion
    GET_POPULAR: 'posts/trend',
```

이렇게만 두는게 어떨까요? 그리고 쿼리쪽에서 처리하는식으로!

Reply:
제가 왜 저기다가 놓았을까요.. 반영할게요 👍